tools/distpartitioning/multi_data_initialize.py

```diff
@@ -0,0 +1,25 @@
+import os
```

Comment:
why do you need a separate file for this function?

Reply:
function is moved to initialize.py and this is removed... as suggested.